docker/dbb_buffmngrs_handoff/Dockerfile

```diff
@@ -0,0 +1,36 @@
+ARG STACK_VER
+FROM lsstsqre/centos:$STACK_VER
+LABEL name="DBB Buffer Manager" \
```

Comment:
This docker container is specific to the handoff machine?   If so, please add to LABEL.